src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -115,6 +115,8 @@ public static class ScriptConstants
         public const string DurableTaskPropertyName = "durableTask";
         public const string DurableTaskHubName = "HubName";
 
+        public const string DevelopmentEnvironmentName = "Development";
```

Comment:
We can keep this or use `Microsoft.Extensions.Hosting.EnvironmentName.Development`